Dockerfile

```diff
@@ -60,7 +60,13 @@ RUN set -ex && apk --update --no-cache add \
     bash \
     libstdc++ \
     libc6-compat \
-    ca-certificates
+    ca-certificates \
+    nodejs \
+    nodejs-npm
+
+# Add TypeScript support
+
+RUN npm i -g ts-protoc-gen google-protobuf
```

Comment:
The latest release of `ts-protoc-gen` is missing `google-protobuf` as a runtime dependency. This was fixed in April (https://github.com/improbable-eng/ts-protoc-gen/pull/161) on master but there hasn't been a new release since January (0.9.0).